code/index.html

```diff
@@ -4,19 +4,174 @@
+          </a>
+        </div>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+    <section class="wrapper-flex-big">
+      <div class="container-big">
+        <img src="./paulista.jpg" alt="Avenida-paulista" class="big-section-img">
+        <div class="conteiner-text">
+          <p class="title-big-news"> São Paulo - The city that never sleeps</p>
+          <p class="text-big-news">Also known as Terra da Garoa, Sampa (São Paulo's nickname) is the largest city in Brazil with 12.2 million inhabitants. It has almost everything you could want on a business or leisure trip. There are many parks around the city, as well as endless options for leisure and culture. It doesn't matter if you prefer cultural tours, museums and theaters, or if you want to do a lot of shopping and taste food from all over the world. São Paulo will not disappoint you!</p>
+        </div>
+      </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <h2 class="section-title">must do's in São Paulo</h2>
+
+    <section class="wrapper-flex">
+      <div class="container">
+        <img src="./Bloco_de carnaval.jpg" alt="Bloco-de-Carnaval" class="photo">
+        <h2 class="title">Let's party</h2>
+        <p class="description">Bloco de Carnaval de Rua -  Street Bands That Mobilize Crowds .</p>
+      </div>
+
+      <div class="container">
+        <img src="./DOM.jpg" alt="DOM-restaurant" class="photo">
+        <h2 class="title">foodie</h2>
+        <p class="description">DOM Restaurant - it is a two Star Michelin, if you are a food lover you will love this place.</p>
+      </div>
+
+      <div class="container">
+        <img src="./vila_madalena.jfif" alt="Vila-Madalena" class="photo">
+        <h2 class="title">brazilian's södermalm</h2>
+        <p class="description">Vila Madalena - has a creative atmosphere with the graffiti  around the area and lots of bars with live music.</p>
+      </div>
+
+      <div class="container">
+        <img src="./ilha_bela.png" alt="ilha-bela" alt="beach" class="photo">
+        <h2 class="title">vitamin sea</h2>
+        <p class="description">São paulo has beautiful beaches - Check it out our favorites.</p> 
+      </div>
+
+      <div class="container">
+        <img src="./terraço-itália-.webp" alt=terraco-italia class="photo" >
+        <h2 class="title">Drink with a view</h2>
+        <p class="description">Check it out, the best rooftop bars in town with breathtaking views from the city.
+        </p> 
+      </div>
+
+      <div class="container">
+        <img src="./bairro_liberdade.jpg" alt="Bairro-Liberdade" class="photo">
+        <h2 class="title">Who likes sushi?</h2>
+        <p class="description">Bairro da Liberdade - The most traditional japonese area.</p>
+      </div>
+
     </section>
+
+    <!-- Here starts the fact section  -->
```

Comment:
I like your comments because they divide the code in smaller parts and it is easier to read :)